### DISCUSSION TITLE
Change components behavior

### DISCUSSION THREAD

**Discussion by awfulminded** [mantinedev/mantine#4786] [Q&A]
### What package has an issue

@mantine/core

### Describe the bug

Is it currently somehow possible to change components behavior? For example remove loading overlay from button and change its spinner to something else?

### What version of @mantine/* page do you have in package.json? (Note that all @mantine/* packages must have the same version in order to work correctly)

/

### Do you know how to fix the issue

None

### Are you willing to participate in fixing this issue and create a pull request with the fix

None

**Comment by rtivital** [comment]
Please use GitHub issues to report bugs, to ask questions, use GitHub discussions. To answer your question – yes, it is possible to change components behavior. If you have more specific questions, create discussions on GitHub or reach out on Discord.
